docs/index.md

```diff
@@ -1,12 +1,52 @@
-# Starter pack
+# Pebble
 
-This starter pack contains the files you need to quickly set up your Sphinx documentation.
+_Take control of your internal daemons!_
 
-Before you start, see the information about `Read the Docs at Canonical`_ and the instructions for `How to publish documentation on Read the Docs`_.
+**Pebble** helps you to orchestrate a set of local service processes as an organized set. It resembles well known tools such as _supervisord_, _runit_, or _s6_, in that it can easily manage non-system processes independently from the system services, but it was designed with unique features that help with more specific use cases.
 
-Then, to start setting up your docs, check the :doc:`ReadMe <readme>` for instructions.
-The `Example product documentation`_ shows how to set up a Diátaxis structure in Sphinx.
+---
 
-For quick help on reST or MyST syntax, see the :file:`doc-cheat-sheet.rst.txt` or :file:`doc-cheat-sheet-myst.md.txt` files in the repository.
-(Open the files in your text editor; the rendered output is not very useful.)
+## In this Documentation
 
+````{grid} 1 1 2 2
+```{grid-item-card} [Tutorials](tutorial/getting-started)
+**Start here**: a hands-on getting started introduction to Pebble for new users.
+```
+
+```{grid-item-card} [How-To Guides](how-to-guides/index)
+**Step-by-step guides** covering key operations and common tasks.
+```
+````
+
+````{grid} 1 1 2 2
+```{grid-item-card} [Explanation](explanation/index)
+**Discussion and clarification** of key topics.
+```
+
+```{grid-item-card} [Reference](reference/index)
+**Technical information** - specifications, APIs, and architecture.
+```
+````
+
+---
+
+## Project and Community
+
+Pebble is free software and released under [AGPL-3.0-only](https://www.gnu.org/licenses/agpl-3.0.en.html).
```

Comment:
I don't think it's AGPL, just GPL 3, right? https://github.com/canonical/pebble/blob/master/COPYING